DTXManiaプロジェクト/コード/スコア、曲/CDTX.cs

```diff
@@ -7111,9 +6997,8 @@ private void t入力_行解析( ref StringBuilder sbコマンド, ref StringBuil
 
 					// オブジェクト記述コマンドの処理。
 
-					else if( !this.t入力_行解析_WAVVOL_VOLUME( strコマンド, strパラメータ, strコメント ) &&
+					else if(
```

Comment:
WAVVOL and VOLUME were specific to DTX files and making it harder to understand what changes needed to be made to properly control sound levels. Nuked.